### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by fabiencastan**
* add compatibility with ilmbase 1.0.1 (the default version on debian based systems)
* fix missing link
* minor cmake clean

### CODE REVIEW

File: cmake/AlembicArnold.cmake [hunk 1]
```diff
@@ -74,11 +74,6 @@ ENDIF()
 FIND_PATH( ALEMBIC_ARNOLD_INCLUDE_PATH ai.h
            PATHS
            "${ALEMBIC_ARNOLD_ROOT}/include"
-           NO_DEFAULT_PATH
-           NO_CMAKE_ENVIRONMENT_PATH
-           NO_CMAKE_PATH
-           NO_SYSTEM_ENVIRONMENT_PATH
-           NO_CMAKE_SYSTEM_PATH
            DOC "The directory where ri.h resides" )
```
Comment (sopvop) [inline cmake/AlembicArnold.cmake]: It's seems 'ri.h' was not changed to 'ai.h' in DOC string after copy-pasting from RenderMan finder.

Comment (fabiencastan) [inline cmake/AlembicArnold.cmake]: Yes, fixed.

**Comment by fabiencastan** [comment]
Is there a problem the PR?

**Comment by rsgalloway** [comment]
Sorry, not necessary. I have not had the time to properly look at this or test it yet. I'll do my best to get to it ASAP.

**Comment by fabiencastan** [comment]
Ok, thanks.

**Pull request merged**